external/emulation/emulation/zhao_carr.py

```diff
@@ -104,7 +104,7 @@ def mask_where_fortran_cloud_identical(state, emulator):
     return _update_with_net_condensation(cloud_out, state, emulator)
 
 
-def _get_classify_output(emulator, one_hot_axis=0):
+def _get_classify_output(emulator, one_hot_axis=0, class_key="gscond_classes"):
```

Comment:
I think this is the right way.  Note that it did require a specification in `ModelWithClassifier` since the usage of "gscond_classes" was implicit with the function call.